frontend/src/components/advanced-auto-filter/advanced-auto-filter.tsx

```diff
@@ -63,6 +64,8 @@ import styles from './styles.module.scss';
 
 const AdvancedAutoFilter: FC = () => {
   const dispatch = useAppDispatch();
+  const [searchParams] = useSearchParams();
+  const brandId = (searchParams.get('brand') as string) ?? '';
```

Comment:
Why do we need the `as string`? `.get` already returns the `string`: 
![image](https://user-images.githubusercontent.com/68850090/189721620-ec4bece2-cdf6-4228-9759-bcae7395d70e.png)

Also, why do we need `??` here instead of the simpler `||`?